feed.xml

```diff
@@ -0,0 +1,25 @@
+---
+layout: null
+---
+
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+  <channel>
+    <title>{{ site.title | xml_escape }}</title>
+    <description>{{ site.description | xml_escape }}</description>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ "/feed.xml" | prepend: site.url }}" rel="self" type="application/rss+xml"/>
+    <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    <generator>Jekyll v{{ jekyll.version }}</generator>
+    {% for page in site.html_pages %}
+      {% unless page.tag == 'omit' %}
+        <item>
+          <title>{{ page.title | xml_escape }}</title>
+          <description>{{ page.content | xml_escape }}</description>
```

Comment:
HI @jeff-matthews small suggestion: use `strip_html`, and `truncatewords: 75` this will improve the load time of the feed. `<description>{{ page.content | xml_escape | strip_html | truncatewords: 75 }}</description>`

Reply:
Thanks @mhauri!